tests/test-cases/Group11-Upgrade/11-01-Upgrade.robot

```diff
@@ -136,6 +142,25 @@ Create Docker Containers
     Wait Until Keyword Succeeds  20x  5 seconds  Hit Nginx Endpoint  %{VCH-IP}  10000
     Wait Until Keyword Succeeds  20x  5 seconds  Hit Nginx Endpoint  %{VCH-IP}  10001
 
+Create Container with Named Volume
+    Log To Console  \nCreate a named volume and mount it to a container\n
+    ${rc}  ${container}=  Run And Return Rc And Output  docker1.11 %{VCH-PARAMS} volume create --name=${namedVolume}
+    Should Be Equal As Integers  ${rc}  0
+    Should Contain  ${container}  ${namedVolume}
+
+    ${rc}  ${output}=  Run And Return Rc And Output  docker1.11 %{VCH-PARAMS} create --name=${mntDataTestContainer} -v ${mntTest} -v ${namedVolume}:${mntNamed} busybox
+    Should Be Equal As Integers  ${rc}  0
+    Set Suite Variable  ${TestContainerVolume}  ${output}
+
+Verify Volume Inspect Info
+    [Arguments]  ${inspectedWhen}  ${volTestContainer}
+    Log To Console  Container Mount Inspected ${inspectedWhen} VCH restart
+    ${rc}  ${mountInfo}=  Run And Return Rc And Output  docker1.11 %{VCH-PARAMS} inspect -f '{{.Mounts}}' ${volTestContainer}
+    Should Be Equal As Integers  ${rc}  0
+    Should Contain  ${mountInfo}  ${mntTest}
+    Should Contain  ${mountInfo}  ${mntNamed}
+    Should Contain  ${mountInfo}  ${namedVolume}
```

Comment:
- [ ] Use Docker-Util.robot's newly created keyword in this file for "Verify Volume Inspect Info"